feat(reference): add dry_run to user creation payload (#51)
ulule/developers.ulule.com: content/reference/user.md

| `email`      | string | Email of the user -- required, max 254 characters                                                                                                                      |
| `lang`       | string | Language of the user -- optional                                                                                                                                       |
| `ip_address` | string | IP address of the user -- optional                                                                                                                                     |

> | `dry_run`    | bool   | If true, fields are checked but not saved in database -- optional                                                                                                      |

Reviewer: sort fields alphabetically

> | `dry_run`    | bool   | If true, fields are checked but not saved in database -- optional                                                                                                      |

Reviewer: ```suggestion
| `dry_run`    | bool   | If true, fields are checked but the user is not created -- optional                                                                                                      |
```
